interfaces/builtin/all.go

```diff
@@ -28,6 +28,7 @@ var allInterfaces = []interfaces.Interface{
 	&BluezInterface{},
 	&BrowserSupportInterface{},
 	&ContentInterface{},
+	NewDcdbasControlInterface(),
```

Comment:
Ideally this would be in the group of those interfaces that start with 'New' down below, but I won't block on this change.